src/hooks/useCalculation.ts

```diff
@@ -0,0 +1,115 @@
+import { useEffect, useState } from 'react';
+import type { UnitBoxes } from '../types';
+import { useStateValue } from '../state/state';
+import { ARDataPriceEstimator } from '../utils';
+
+/** ARDataPriceEstimator instance, will fire off initial fetch calls when constructed */
+const arDataPriceEstimator = new ARDataPriceEstimator();
+
+const winstonToArValue = 0.000_000_000_001;
```

Comment:
Why not create a constants file for all the constants?

Reply:
As long as they're all in the same domains. Otherwise it could get challenging to reuse code later on with first having to carve out or duplicate constants.